tests/unit/cluster/links.tcl

```diff
@@ -161,7 +216,6 @@ start_cluster 3 0 {tags {external:skip cluster}} {
         } else {
             fail "Cluster link not freed as expected"
         }
-        puts -nonewline "$i 128KB messages needed to overflow 256KB buffer limit. "
```

Comment:
why remove this?

Reply:
This isn't helpful, seems like a stray debug statement, hence removed.

Reply:
This was part of some earlier debugging code because freeBSD was failing oddly on the old cluster links test framework.